### PR TITLE
Added fortnite store app

### PR DESCRIPTION
**Pull request by naomi-nori**
A take on a fortnite store app that pulls the current for-sale items and randomly selects one to display.
fortnite_store

### CODE REVIEW

**Comment by naomi-nori** [comment]
I have read the CLA Document and I hereby sign the CLA

File: apps/fortnitestore/fortnite_store.star [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
...bw/M1gR9cubtlPgvg15a61dAxVwBYCIODB3BqegprW9vQ07OtLvlebCLArXf14tAry5sDoJ4AMDw8hD0DT/gEgA9u/wXEkyxJXyMhoJ71305ASCuX6RYL1gQwUN7OFgEuivxfFzOORVyjUCDzf/CeW9Z/0mRl416a9R8GhH/hIwvHdUr+P4VQ6K+0uVnoxdnafaoouiKAqoJSykqgXqgIYJF/33ksYP0PHssm/zVzANiWf00DNYlD/i817HMIhUKjvGVxcAGAg4NjWbG2ZeW317ZsvOml4mlgLofXzrnW19TGzWi/owdPPvW0PXSjpFPk5xirjhWBR9+9BUjFMNg/jMH+ITz1gxwAGdE1bPIsrPYT9ojfYI8zZb8L+qmAR/qh14ImzzLkNe4EPtU0i9QqdzkW9ludg1UI0uk80mlg9qyO8akSiqUUplt3AbEuRIvj2GrMAqoKVYlD1YBYPGYSWUY0noSqqpivhH58Oe+PNz/D3qH/8SyUxK/lSzOOdT2eSqJcLkFVS9YdCkwiZAAicQoqkoAxQfIUWjx0vB/PjP45soUpIAOcKn0MH3vwc87vhZkMZDmOx3b9Ib7+nf+IgdefQ99PPYxSIQ9dZTfOmwwpIsvIzeaRPZPHbD6PQiEPuTnqzzdg3aCoJKMkx5FKxbAqqiAyyxqKqgLySoDqgK4DXTfeis3vuhdqjfIS/T/+Lo6cPghpBftPR/7tIIpkFo8/9BV2PtIKUGpA9+QhkCPWdZEEqFAhCBokCRDl2kJA0PtTm7e+sHikau+6YoA0qlO92Emg6bmjBFiy474QiDkN7MAkVew10ADt0AN7w1C1JNbw/C/6CbjIC0jcXdh/wS77FSLu/3XKA9LAfQ+QfxEgEQmisoL9nowhEo0iqjBPplubNuz9eDL5449buTu8zy8Hx7WM+bNnviwa+hf0s7PA3BmgOGuVsrOZtAaiApTqIIZl+ScEhBAAGgCjKmcJaDCnSaBHDVjmqSUkGJ717W0Mw/1sk2taZrZ4Yn2vqhbZpxS6YYDOG462QE3irKdbI61mOgdixwh2VyFa1Z0xh4PaI0PZdP8PUWKQ5BaY4VU/4K2LgwsAHBwcy4Y/ffIHm/7wD75+07Zt9wAADhw4iE6PCX5mYgSp1FpsvmMTnnzqaWy+fRPSrYz8Z07lkW6NoaePFVwf7B/CYP8womtYHfbmFkb
```
Comment (matslina) [inline apps/fortnitestore/fortnite_store.star:38]: The secret doesn't seem to work as intended in our production environment. Could you share the command line you used to generate it (excluding the plaintext secret itself of course)?

Comment (matslina) [inline apps/fortnitestore/fortnite_store.star:33]: This isn't a huge issue, but if you scale that graphic down a bit before base64 encoding we'll get a bit less data in the .star file which is nice when reviewing. =)

Comment (matslina) [inline apps/fortnitestore/fortnite_store.star:59]: Would be great to add some caching around this http call as well. Just to make sure we're not unnecessarily refetching the same image over and over again.

**Review by matslina — CHANGES_REQUESTED** [review]
Hey @naomi-nori and thanks for the PR! Looks great. Just a few issues before we can merge. In particular the secrets problem needs to be figured out, as it's currently always failing to decrypt.

File: apps/fortnitestore/fortnite_store.star (hunk 1 above)
Comment (betterengineering) [reply apps/fortnitestore/fortnite_store.star]: I updated this app to use our own secret

Comment (betterengineering) [reply apps/fortnitestore/fortnite_store.star]: I added caching to the image

**Review by betterengineering — APPROVED** [review]
Hey @naomi-nori! I've added a few changes to get this over the finish line since it's been open for so long. Thanks for the app!

**Pull request merged**